src/config/app.ts

```diff
@@ -182,6 +210,31 @@ export const getSpecterWalletConfig = (): SpecterWalletConfig => {
   }
 }
 
+type TwilioConfig = {
```

Comment:
Can maybe place this in `src/core/types.d.ts` to be consistent with all types being in declaration files and injected into global environment

Reply:
we should not add more code to the core folder. Maybe a new type file in config folder?